python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt_tests/dlt_tests/test_dlt.py

```diff
@@ -0,0 +1,160 @@
+import os
+from typing import Optional
+
+import dlt
+import duckdb
+import pytest
+from dagster import (
+    AssetExecutionContext,
+    AssetKey,
+    AutoMaterializePolicy,
+    AutoMaterializeRule,
+    Definitions,
+)
+from dagster._core.definitions.auto_materialize_rule import MaterializeOnCronRule
+from dagster._core.definitions.materialize import materialize
+from dagster_embedded_elt.dlt.asset_decorator import dlt_assets
+from dagster_embedded_elt.dlt.resource import DagsterDltResource
+from dagster_embedded_elt.dlt.translator import DagsterDltTranslator
+from dlt.extract.resource import DltResource
+
+from .dlt_test_sources.duckdb_with_transformer import pipeline
+
+EXAMPLE_PIPELINE_DUCKDB = "example_pipeline.duckdb"
+
+DLT_SOURCE = pipeline()
+DLT_PIPELINE = dlt.pipeline(
+    pipeline_name="example_pipeline",
+    dataset_name="example",
+    destination="duckdb",
+)
+
+
+@pytest.fixture
```

Comment:
To remove the need to specify `_teardown` on all your tests

```suggestion
@pytest.fixture(autouse=True)
```